.gitignore

```diff
@@ -7,6 +7,9 @@ pids
 *.pid
 *.seed
 
+# Configuration
+config.json
```

Comment:
We won't be using configurator for this repository as it's public. The tests require a username and api key, but I think it's wrong to impose an internal config structure on a public repository.

I specifically chose to use `config.json` instead of `configuration.json` to move away from configurator.